### PR TITLE
New profile menu drop down

### PR DESCRIPTION
**Pull request by tiffwong**
## Description
- [ ] Links in the menu should match the corresponding names used in the tabs
- [ ] Links in the menu should match the order the tabs show up on the corresponding pages
- [x] Account should be hidden for restricted users

### CODE REVIEW

File: packages/manager/src/features/TopMenu/UserMenu/UserMenu_CMR.tsx [hunk 1]
```diff
@@ -187,60 +183,86 @@ export const UserMenu: React.FC<{}> = () => {
...
+            <div className={classes.menuHeader}>My Profile</div>
+            <Grid
+              container
+              wrap="wrap"
+              direction="column"
+              className={classes.profileWrapper}
+            >
+              {profileLinks.map(renderLink)}
+            </Grid>
+            {_isRestrictedUser ? null : (
```
Comment (johnwcallahan) [inline packages/manager/src/features/TopMenu/UserMenu/UserMenu_CMR.tsx:245]: Technically this is slightly more complicated... a restricted user could have `read_only` account access, in which case they can view billing info and account settings (but not users). A restricted user with `read_write` account access can modify users as well (still not users, though).

**Review by johnwcallahan — CHANGES_REQUESTED** [review]
This looks nice, great job. One comment about restricted users, and also the "Log out" link strangely sits outside of the container on FireFox:
Screen Shot 2020-12-03 at 11 27 45 AM

**Review by johnwcallahan — APPROVED** [review]
Pushed up a typo fix, LGTM.

**Review by DevDW — APPROVED** [review]
Looks nice and things are working, maybe add a `cursor: pointer;` on the notification icon

**Pull request merged**